docs/admin/runai-setup/access-control/rbac.md

```diff
@@ -73,13 +74,14 @@ A *Scope* is an organizational component which accessible based on assigned role
 
 ### RBAC enforcement
 
-RBAC ensures that user have access to system assets based on the rules that are applied to those assets. Should an asset be part of a larger scope of assets to which the user does not have access. The scope shown to the user will appear to be incomplete because the user is able to access **only** the assets to which they are authorized.
-
+RBAC ensures that user have access to system assets based on the access rules that are applied to those assets. <!-- Not sure I understand the sentense that start after this comment and the reason it was added. What is the point we are trying to make or action that we expect the user to take? --> Should an asset be part of a larger scope of assets to which the user does not have access. The scope shown to the user will appear to be incomplete because the user is able to access **only** the assets to which they are authorized.
```

Comment:
Remove in line 77 the first comment to the end of sentence till the use case is verified.